.travis.yml

```diff
@@ -55,6 +53,7 @@ jobs:
         - ./scripts/if_changed.sh ./scripts/pod_install.sh
       script:
         - ./scripts/if_changed.sh ./scripts/build.sh $PROJECT $PLATFORM $METHOD
+        - ./scripts/if_changed.sh bundle exec pod lib lint FirebaseFirestore.podspec --allow_warnings
```

Comment:
I'm willing to bet that you could skip the `pod_install.sh` step in your task, something like 

```
    - stage: test
      env:
        - PROJECT=Firestore PLATFORM=iOS METHOD=pod-lib-lint
      before_install:
        - ./scripts/if_changed.sh ./scripts/install_prereqs.sh
      script:
        - ./scripts/if_changed.sh bundle exec pod lib lint FirebaseFirestore.podspec --allow_warnings
```